resources/agent.rb

```diff
@@ -36,8 +36,32 @@
                                    "backend-url": ['ws://127.0.0.1:8081'],
                                  }
 action :install do
+  if platform?('windows')
+    include_recipe 'chocolatey'
+    chocolatey_package 'sensu-agent' do
```

Comment:
May need someone to confirm that this package is produced and managed as an official release: https://chocolatey.org/packages/sensu-agent

@cwjohnston maybe?

Reply:
Confirmed! These are the official Windows packages published by us. 👍